tests/test_subreddit.py

```diff
@@ -82,10 +81,8 @@ def test_get_subreddit_recommendations(self):
 
     @betamax()
     def test_multiple_subreddit__fetch(self):
-        with warnings.catch_warnings(record=True) as w:
-            self.r.get_subreddit('python+redditdev', fetch=True)
-            assert len(w) == 1
-            assert isinstance(w[0].message, UserWarning)
+        self.assertWarnings(UserWarning, self.r.get_subreddit,
```

Comment:
Oops. I mistakenly assumed that was one of the features added to UnitTest that is only supported in 2.7+.